packages/twitch-pubsub-client/src/Messages/PubSubSubscriptionMessage.ts

```diff
@@ -4,20 +4,22 @@ import TwitchClient from 'twitch';
 
 export interface PubSubSubscriptionDetail {
 	context: 'sub' | 'resub';
+	'cumulative-months': number;
+	'streak-months': number;
 }
 
 export interface PubSubSubscriptionGiftDetail {
-	context: 'subgift';
+	context: 'subgift' | 'anonsubgift';
 	recipient_id: string;
 	recipient_user_name: string;
 	recipient_display_name: string;
+	months: number;
```

Comment:
Does this property _always_ exist?

Reply:
From looking at the docs it always exists for gift subs, but I will ask in the discord.

Reply:
From the discord I got told it should be there.